packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -290,7 +292,7 @@ class _CodeViewState extends State<CodeView>
 
     return HistoryViewport(
       history: widget.controller.scriptsHistory,
-      generateTitle: (script) => script.uri,
+      generateTitle: (dynamic script) => script.uri,
```

Comment:
avoid ever adding dynamic to work around null safety issues.

Reply:
Oops! I missed removing these from the null safety tool auto-migration